File: src/main/java/com/p2p/lending/dao/BankcardDao.java
package com.p2p.lending.dao;

import java.util.List;
import java.util.Map;

import com.p2p.lending.pojo.Approveitem;
import com.p2p.lending.pojo.Bankcard;

public interface BankcardDao {
	List<Bankcard> selectbc(Map<String, Object> map);
	int bankcount(Map<String, Object> map);
}

File: src/main/java/com/p2p/lending/repository/BankcardDao.kt
package com.p2p.lending.repository

import com.p2p.lending.entity.Bankcard
import org.springframework.stereotype.Repository

@Repository
interface BankcardDao {

    fun selectbc(map: Map<String, Any?>): List<Bankcard>

    fun bankcount(map: Map<String, Any?>): Int
}
